### PR TITLE
feat: Update navbar and dropdown

### PR DESCRIPTION
**Pull request by Manavpatel2005**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import React from "react";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-import NavigationBar from "./Components/Navbar";
+import { NavigationBar } from "./Components/Navbar/index";
```
Comment (vaidehi27) [inline src/App.tsx]:
```suggestion
import { NavigationBar } from "./Components/Navbar";
```

File: src/App.tsx [hunk 2]
```diff
@@ -12,6 +16,7 @@ export const App: React.FunctionComponent = () => {
         <Routes>
           <Route caseSensitive path="/" element={<Home />} />
           <Route caseSensitive path="/festival" element={<FestivalPage />} />
+          <Route caseSensitive path="/Contact" element={<Contact />} />
```
Comment (vaidehi27) [inline src/App.tsx]:
```suggestion
          <Route caseSensitive path="/contact" element={<Contact />} />
```

File: src/Pages/Festival/constant.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
-import Janmashtami from "../../assets/image/janmashtami.jpg";
-import RathYatra from "../../assets/image/rathYatra.jpg";
-import Holi from "../../assets/image/holi.jpg";
-import Radhashtami from "../../assets/image/radhashtami.jpg";
-import GovardhanPuja from "../../assets/image/govardhanPuja.jpg";
-import KartikDipawali from "../../assets/image/kartikDipawali.jpg";
+import Janmashtami from "../../assets/images/janmashtami.jpg";
```
Comment (vaidehi27) [inline src/Pages/Festival/constant.ts]:
```suggestion
import janmashtamiImage from "../../assets/images/janmashtami.jpg";
```

File: src/Components/Dropdown/style.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import { Link } from "react-router-dom";
+
+// export const TransparentDropdown = styled.div`
+//   display: inline-block;
+//   position: relative;
+//   z-index: 1;
+//   background: transparent;
+//   border: none;
+// `;
+
```
Comment (vaidehi27) [inline src/Components/Dropdown/style.ts]:
```suggestion
```

File: src/Components/Dropdown/style.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+`;
+
+export const DropdownMenu = styled.div`
+  border: none;
+  box-shadow: none;
+  background-color: white;
+`;
+
+export const DropdownToggle = styled.div`
+  color: red;
```
Comment (vaidehi27) [inline src/Components/Dropdown/style.ts]: use color code

File: src/Components/Navbar/constant.ts [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    href: "https://instagram.com/iskcon_vallabh_vidhyanagar",
+    icon: "instagram",
+  },
+  {
+    name: "youtube",
+    href: "https://www.youtube.com/channel/UCkxYEp2omi0yV3LcutvQJ-w",
+    icon: "youtube",
+  },
+];
+export const dropdownItems = [
```
Comment (vaidehi27) [inline src/Components/Navbar/constant.ts]:
```suggestion
export const dropdownHomeInfo = [
```

File: src/Components/Navbar/style.ts [hunk 7]
```diff
@@ -1,12 +1,61 @@
...
+    color: #7e5959;
+    background-color: transparent;
+  }
+
+  .dropdown-menu {
+    background-color: #7e5959;
+  }
+
+  .dropdown-item {
+    color: white;
```
Comment (vaidehi27) [inline src/Components/Navbar/style.ts]: use color code

File: src/Components/Navbar/index.tsx [hunk 8]
```diff
@@ -1,26 +1,41 @@
...
+      <nav className="navbar">
+        <div className="container">
+          <DropDown items={dropdownHomeInfo} dropdownTitle="ISKCON" />
+          <DropDown items={dropdownFestivalInfo} dropdownTitle="Festival" />
+          <DropDown items={dropdownServicesInfo} dropdownTitle="Services" />
+          <StyledLink to="/">
+            <ImgStyle src={logoUrl} alt="Logo" />
+          </StyledLink>
+          <DropDown items={dropdownMediaInfo} dropdownTitle="media" />
+          <StyledLink to="Contact">Contact</StyledLink>
```
Comment (vaidehi27) [inline src/Components/Navbar/index.tsx]:
```suggestion
          <StyledLink to="/contact">Contact</StyledLink>
```

File: src/Components/Dropdown/index.tsx [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+  title: string;
+  path: string;
+}
+
+interface DropdownProps {
+  items: DropdownItem[];
+  toggleTitle: string;
+}
+
+export const DropDown: React.FC<DropdownProps> = ({ items, toggleTitle }) => {
```
Comment (vaidehi27) [inline src/Components/Dropdown/index.tsx]:
```suggestion
export const Dropdown: React.FC<DropdownProps> = ({ items, toggleTitle }) => {
```

File: src/Components/Dropdown/style.ts [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+    font-size: 1rem;
+  }
+`;
+
+export const DropdownToggleStyle = styled.div`
+  color: #7e5959;
+  background-color: transparent;
+`;
+
+export const DropdownMenuStyle = styled.div`
```
Comment (vaidehi27) [inline src/Components/Dropdown/style.ts]:
```suggestion
export const DropdownMenuWrapper = styled.div`
```

File: src/Components/Navbar/index.tsx [hunk 11]
```diff
@@ -1,64 +1,35 @@
...
-          </nav>
-        </NavLink>
-      </div>
+      <nav className="navbar fixed-top">
+        <div className="container">
+          <PageDropdown items={dropdownHomeInfo} toggleTitle="ISKCON" />
+          <PageDropdown items={dropdownServicesInfo} toggleTitle="Services" />
+          <StyledLink to={routes.visitUs}>Visit Us</StyledLink>
+          <StyledLink to={routes.media}>Media</StyledLink>
+          <StyledLink to="/">
```
Comment (vaidehi27) [inline src/Components/Navbar/index.tsx]:
```suggestion
          <StyledLink to={routes.home}>
```

File: src/Components/Navbar/style.ts [hunk 12]
```diff
@@ -1,12 +1,60 @@
 import styled from "styled-components";
+import title from "../../assets/images/navbar-bg.png";
+import { Link } from "react-router-dom";
 
 export const NavStyle = styled.div`
```
Comment (vaidehi27) [inline src/Components/Navbar/style.ts]:
```suggestion
export const NavStyle = styled.nav`
```
styles can be applied directly to nav component

File: src/Components/Navbar/style.ts [hunk 13]
```diff
@@ -1,12 +1,60 @@
...
+  @media (min-width: 1400px) {
+    height: 85px;
+  }
+`;
+export const SocialMediaIconsStyle = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  a {
```
Comment (vaidehi27) [inline src/Components/Navbar/style.ts]: why nested `a` tag?

File: src/Components/Navbar/style.ts (hunk 12 above)
Comment (Manav2304) [reply src/Components/Navbar/style.ts]: If I make changes, the navbar background is not being applied

File: src/Components/Navbar/style.ts (hunk 13 above)
Comment (Manav2304) [reply src/Components/Navbar/style.ts]: i used nested <a> tag for style social media icon

**Review by vaidehi27 — APPROVED** [review]

**Pull request merged**